Makefile

```diff
@@ -1,6 +1,5 @@
 .PHONY: build
 build:
-	pipx install cibuildwheel
```

Comment:
This was unnecessary. Leftover because my misunderstanding the relationship between `cibuildwheel` and `build`.

You don't need `cibuildwheel` to run `build`.